config/routes.rb

```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
+  get 'users/new'
```

Comment:
you don't need this, just remove it

Reply:
Remove means delete it or move it to .gitignore?

Reply:
the book ask me to put this sentence, is it okay to remove it?

Reply:
Done